varianteval/scripts/build_igv_batch_script.py

```diff
@@ -0,0 +1,81 @@
+"""
+Script to write IGV batch scripts with instructions to screenshot each record from an input vcf
+"""
+import numpy as np
+# import sys
+from pysam import VariantFile
+# import os
+import argparse
+
+
+def generate_script(input_vcf, bam_path, output_batch_script_path, igv_screenshot_dir, min_svlen,
+                    max_svlen, trg_supp_vec=None, genome='hg38', colorby_ins_size=True,
+                    groupby_pair_orientation=True, viewaspairs=True):
+    """
+    method to generate a batch script for an input vcf
+    """
+    out = open(output_batch_script_path, 'w')
+    out.write('new\n')
+    out.write(f'genome {genome}\n')
+    out.write('preference SAM.MAX_VISIBLE_RANGE 1000\n')
+    out.write('preference SAM.SHOW_MISMATCHES FALSE\n')
+    out.write(f'load {bam_path}\n')
+
+    write_first_time = True
+    input_vcf_file = VariantFile(input_vcf)
+    for rec in input_vcf_file.fetch():
+        svlen = rec.info['SVLEN']
```

Comment:
done

Reply:
deleting this filter here is not needed, just switch to getting the length from the record (instead of relying on the SVLEN field which might be missing); currently the filtering params are still in the cli and passed to the function but not used

Reply:
Of course, my mistake -- done